frontend/src/components/page_content/action-items-content.js

```diff
@@ -1,11 +1,20 @@
 import React from "react";
 import Style from "style-it";
 
-import ToastCheckbox from "../toast/toast-checkbox";
+import EmergencySavings from "../page_content/action-items/emergency-savings";
+import Protection from "../page_content/action-items/protection";
+import Debt from "../page_content/action-items/debt";
+import Retirement from "../page_content/action-items/retirement";
+import Budgeting from "../page_content/action-items/budgeting";
+
+import { resetStep } from "../../redux/actions";
 
 import { connect } from "react-redux";
 
 class ActionItemsContent extends React.Component {
+  componentDidMount() {
```

Comment:
i dont think this is necessary because its reset already in the page template component